spec/unit/matrixrtc/mocks.ts

```diff
@@ -60,8 +60,7 @@ export function mockRTCEvent(
         }),
         getSender: jest.fn().mockReturnValue("@mock:user.example"),
         getTs: jest.fn().mockReturnValue(1000),
-        getLocalAge: getLocalAgeFn,
-        localTimestamp: Date.now(),
+        localTimestamp: Date.now() - getLocalAgeFn(),
```

Comment:
why not pass in `localAge`, rather than a function `getLocalAge` which is evaluated immediately?

Reply:
This would mean chaning a a couple of tests whcih use this mock fn. But this is also makes sense to me. its just a bigger refactor.

Reply:
I refactored it to use a property instead of a function. It is not possible anymore to update the expiration time because the localTImestamp is now a fixed value.
So we would need to make sure that Date.now() is not called when "remocking" the rtc event which happens when`getStateEvents` is called.
Therefore the the local Timestamp is already computed in `makeMockRoom` which is only called once per test and not indirectly by the code we want to test and is passed over to `getStateEvents`.

This is much better anyways since we sometimes rely on the `localTimestamp` and sometimes on the `getLocalAge` fn (but `getLocalAge` directly relies on the `localTimestamp` so forgetting to mock both in sync would always have lead to issues. Now mocking the `localTimestamp` give is indirectly mocking all other age related methods as well.

Reply:
yes this is all much nicer. Thanks for taking the time to do this!